Library/Homebrew/cmd/update.rb

```diff
@@ -398,10 +398,26 @@ def select_formula(key)
 
   def dump_formula_report(key, title)
     formula = select_formula(key)
+    formula_installed = select_installed_formula(key, formula)
     formula.map! { |oldname, newname| "#{oldname} -> #{newname}" } if key == :R
     unless formula.empty?
       ohai title
-      puts_columns formula
+      puts_columns(formula, formula_installed)
```

Comment:
Have you been reading my mind? ;) I'd like to have `Tty` color support in `puts_columns`, but that's not a trivial change because `puts_columns` is using an external command (`pr`) for the multi-column output. And `pr` doesn't play nicely with ANSI escape sequences.

I have a working prototype and would prefer to contribute it in a separate PR, because it's completely independent of this change.